signingscript/src/signingscript/utils.py

```diff
@@ -210,7 +210,7 @@ def is_apk_autograph_signing_format(format_):
 
     """
     # TODO Remove autograph_focus once format is migrated
-    return format_ and format_.startswith("autograph_apk_") or format_ == "autograph_focus"
+    return format_ and format_.startswith("autograph_apk_") or format_ == "autograph_focus" or format_ == "autograph_stage_aab"
```

Comment:
Why didn't we need to handle the `autograph_stage_apk_v3` case here?

Reply:
Good question! I think we've never handled v3 signing in signingscript. Hal Wine and I had a conversation about it before the all hands. I'll try to add support if I find some time in the next week and if it doesn't conflict the work on AAB. The latter remains our priority.